kayak_core/src/styles/mod.rs

```diff
@@ -138,22 +145,40 @@ define_styles! {
     pub struct Style {
         pub background_color : StyleProp<Color>,
         pub border_color: StyleProp<Color>,
-        pub border_radius: StyleProp<(f32, f32, f32, f32)>,
-        pub border: StyleProp<(f32, f32, f32, f32)>,
+        pub border_radius: StyleProp<Corner<f32>>,
+        pub border: StyleProp<Edge<f32>>,
         pub bottom: StyleProp<Units>,
         pub color: StyleProp<Color>,
+        /// The spacing between child widgets along the horizontal axis
+        pub col_between: StyleProp<Units>,
         pub cursor: StyleProp<CursorIcon>,
         pub height: StyleProp<Units>,
         pub layout_type: StyleProp<LayoutType>,
         pub left: StyleProp<Units>,
-        pub margin_bottom: StyleProp<Units>,
-        pub margin_left: StyleProp<Units>,
-        pub margin_right: StyleProp<Units>,
-        pub margin_top: StyleProp<Units>,
         pub max_height: StyleProp<Units>,
         pub max_width: StyleProp<Units>,
         pub min_height: StyleProp<Units>,
         pub min_width: StyleProp<Units>,
+        /// The positional offset from this widget's default position
+        ///
+        /// This property has lower precedence than its more specific counterparts
+        /// ([`top`](Self::top), [`right`](Self::right), [`bottom`](Self::bottom), and [`left`](Self::left)),
+        /// allowing it to be overridden.
+        ///
+        /// For widgets with a [`position_type`](Self::position_type) of [`PositionType`](PositionType::ParentDirected)
+        /// this acts like margin around the widget. For [`PositionType`](PositionType::SelfDirected) this
+        /// acts as the actual position from the parent.
+        pub offset: StyleProp<Edge<Units>>,
```

Comment:
Let me know if this should be called something else. I opted for `offset` over `position` since it can act like either margin or position depending on the `position_type`.

Reply:
offset is fine 👍